media_management_api/media_service/tests/test_views.py

```diff
@@ -220,6 +220,24 @@ def test_update_course(self):
             self.assertEqual(response.data[f], body[f])
             self.assertEqual(getattr(course_after_update, f), body[f])
 
+    def test_course_images_list(self):
+        self.client.force_authenticate(self.superuser)
+
+        pk = 1
+        url = reverse('api:course-images', kwargs={"pk": pk})
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(response.json()[0]['url'], 'http://testserver/api/images/1')
+
+    def test_course_images_csv(self):
+        self.client.force_authenticate(self.superuser)
+
+        pk = 1
+        url = reverse('api:course-images-csv', kwargs={"pk": pk})
+        response = self.client.get(url)
```

Comment:
It might be good to sanity check the CSV response and confirm that the header row is present and that it includes the expected fields. Also similar to `test_course_images_list` maybe check that the first row's `url` is what we would expect.

Reply:
Makes sense, sure thing!